Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs

```diff
@@ -8,6 +8,7 @@ public class SkyboxController : MonoBehaviour
     [HideInInspector]
     public int SecondsInDay = 86400;
     public bool PlayOnStart;
+    public bool StopRefresh = false;
```

Comment:
Perhaps it should be a property? You dont care about unity's serialization, right?
Also, i think its better to rename it to `RefreshEnabled` and reverse the conditional, so we can remove negations and its easier to follow.

Reply:
this is only for shibu to be able to edit. I need to refactor the whole thing to be a system in the future